build/xtask/src/humility.rs

```diff
@@ -24,6 +25,7 @@ pub fn run(
     let archive = Path::new("target")
         .join(&toml.name)
         .join("dist")
+        .join(image_name)
```

Comment:
It would be good to check the `image_name` against the `app.toml` before calling Humility, since the failure mode would be confusing (Humility would complain about a missing archive, I think?)

Reply:
Yes, good idea.